joint_trajectory_controller/test/joint_trajectory_controller_test.cpp

```diff
@@ -1244,6 +1244,17 @@ TEST_F(JointTrajectoryControllerTest, pathToleranceViolation)
   EXPECT_TRUE(waitForState(action_client, SimpleClientGoalState::ABORTED, long_timeout));
   EXPECT_EQ(action_client->getResult()->error_code, control_msgs::FollowJointTrajectoryResult::PATH_TOLERANCE_VIOLATED);
 
+  // Check that we're not moving
+  StateConstPtr state1 = getState();
+  ros::Duration(0.5).sleep(); // Wait
```

Comment:
This might be a little bit too high for the hold trajectory  to be effective, but with the position plausibility checks this does not matter (apart from longer test time..)

Reply:
I made this consistent with the other tests. See my other reply.

Reply:
As long as system time is used, it is difficult to check that the robot stopped in time.
Because there can be delay in the state-publisher.